### ISSUE TITLE
Display is broken with some expression

### ISSUE THREAD

**Issue by rmeniche** [arnog/mathlive#524]
## Prerequisites

-   [:+1: ] New issue

## Description
Hello, 

When you type an expression like x²*4 for example, there is a problem, the expression is deleted and only the * stays. I guess that a shortcut is causing trouble.

### Actual behavior

![](https://i.imgur.com/lHMj6gx.png)

### Expected Behavior

![](https://i.imgur.com/w0fHZQz.png)

## Steps to Reproduce

1- Go to mathlive.io
2- Type x ^ 2 * quickly

(Same when you do x ^ *, x ^ 3 * etc...)

### Environment

AZERTY Keyboard

**Operating System**
Windows 10

**Browser** 
Firefox

http://mathlive.io/

**Comment by arnog** [comment]
I have been unable to reproduce this. 

Could you try with Chrome?
Does this happen on macOS or Linux as well?
Does this happen if you switch your keyboard to QWERTY? 
Does this happen if you only have the "*", i.e. "2*4", or does the "*" need to be after a superscript?

**Comment by rmeniche** [comment]
Okay, sorry, I wasn't able to work on this, I'm going to test what I can do and tell you later

**Comment by arnog** [comment]
Great, thanks.

**Comment by rmeniche** [comment]
Hello

So I've asked two persons to test with me.

- One tested with linux, there was no problem for him
- One tested with Chrome, same
- I've tested with Edge, same
- The problem is still here with Mozilla

" Does this happen if you only have the "", i.e. "24", or does the "*" need to be after a superscript?"

I typed 24* and no problems happened.
I typed x^2/ and no problems happened
I typed 1_2* and no problems happened

So I guess it needs to be after a superscript.

Plus, this can happen even if i type ^2 then wait 30sec then type * (there seems to be a limit because 2 min does not create this problem)
